test/Infrastructure.IntegrationTest/AdminConsole/Repositories/OrganizationUserRepository/OrganizationUserRepositoryTests.cs

```diff
@@ -724,4 +724,67 @@ public async Task CreateManyAsync_WithCollectionAndGroup_SaveSuccessfully(
         Assert.Equal(collection3.Id, orgUser3.Collections.First().Id);
         Assert.Equal(group3.Id, group3Database.First());
     }
+
+    [DatabaseTheory, DatabaseData]
+    public async Task ReplaceAsync_PreservesDefaultCollections_WhenUpdatingCollectionAccess(
+        IUserRepository userRepository,
+        IOrganizationRepository organizationRepository,
+        IOrganizationUserRepository organizationUserRepository,
+        ICollectionRepository collectionRepository)
+    {
+        // Arrange
```

Comment:
Thanks for putting the arrange, act, and assert. It makes things a lot easier to follow.